src/applications/vaos/new-appointment/index.jsx

```diff
@@ -73,200 +66,116 @@ export function NewAppointment() {
+          <TypeOfAudiologyCarePage />
+        </Route>
+        <Route path={`${match.url}/preferred-date`}>
+          {featureUseVaDate ? (
+            <PreferredDatePageVaDate />
+          ) : (
+            <PreferredDatePage />
+          )}
+        </Route>
+        <Route path={`${match.url}/date-time`}>
+          <DateTimeSelectPage />
+        </Route>
+        <Route exact path={`${match.url}/va-request/`}>
+          <VARequest />
+        </Route>
+        <Route exact path={`${match.url}/community-request/`}>
+          <CCRequest />
+        </Route>
+        <Route path={`${match.url}/location`}>
+          <VAFacilityPageV2 />
+        </Route>
+        <Route path={`${match.url}/provider`}>
+          <ProviderSelectPage />
+        </Route>
         <Route
           path={`${match.url}/how-to-schedule`}
           component={ScheduleCernerPage}
         />
         <Route
-          path={`${match.url}/community-care-preferences`}
-          component={CommunityCareProviderSelectionPage}
-        />
+          path={[
+            `${match.url}/va-request/community-care-preferences`,
+            `${match.url}/community-request/preferred-provider`,
+          ]}
+        >
+          <CommunityCareProviderSelectionPage />
+        </Route>
         <Route
-          path={`${match.url}/community-care-language`}
-          component={CommunityCareLanguagePage}
-        />
+          path={[
+            `${match.url}/va-request/community-care-language`,
+            `${match.url}/community-request/preferred-language`,
+          ]}
+        >
+          <CommunityCareLanguagePage />
+        </Route>
         <Route
-          path={`${match.url}/choose-closest-city`}
-          component={ClosestCityStatePage}
-        />
-        <Route path={`${match.url}/clinics`} component={ClinicChoicePage} />
+          path={[
+            `${match.url}/va-request/choose-closest-city`,
+            `${match.url}/community-request/closest-city`,
+          ]}
+        >
+          <ClosestCityStatePage />
+        </Route>
+        <Route path={`${match.url}/clinic`}>
+          <ClinicChoicePage />
+        </Route>
         <Route
-          path={`${match.url}/reason-appointment`}
-          component={ReasonForAppointmentPage}
-        />
-        <Route path={`${match.url}/review`} component={ReviewPage} />
+          path={[
+            `${match.url}/va-request/reason`,
+            `${match.url}/community-request/reason`,
+            `${match.url}/reason`,
+          ]}
+        >
+          <ReasonForAppointmentPage />
+        </Route>
+        <Route
+          path={[
+            `${match.url}/va-request/review`,
+            `${match.url}/community-request/review`,
+            `${match.url}/review`,
+          ]}
+        >
+          <ReviewPage />
+        </Route>
         <Route
           path={`${match.url}/confirmation`}
           component={ConfirmationPage}
         />
```

Comment:
While self closing `<Route />`  tag is completely valid, let's keep things consistent by using the opening and closing `<Route></Route>` like the other patterns

Reply:
@ryanshaw I was thinking the pattern more like
```
        <Route path={`${match.url}/confirmation`}>
          <ConfirmationPage />
        </Route>
```